README.md

```diff
@@ -45,6 +45,28 @@ Start the speed test:
 npm start
 ```
 
+### Other scripts
+
+#### Check if a DAG is complete in S3
+
+1. Read all CAR files pertaining to the CID from S3
+2. Add the blocks to an in memory blockstore
+3. Walk the DAG to ensure completeness
+
+```sh
+node is-car-dag-complete.js bafybeih6qj6w3kfesrhy5gxo2lvph2p4uvpl7sq3u2rorvjaooqngqwa5y
```

Comment:
if possible it would be nice to know a sample cid that should fail this check (but afaik we haven't actually encountered any in the wild)